app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1558,8 +1734,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
     
     private void createPreferencesMenu() {
-        Intent i = new Intent(this, CommCarePreferences.class);
-        CommCareHomeActivity.this.startActivityForResult(i, PREFERENCES_ACTIVITY);
+        createPreferencesMenu(this);
+    }
+
+    public static void createPreferencesMenu(Activity activity) {
```

Comment:
Damn, didn't realize how hard it would be to make some of these changes! I bet some of my comments are moot since I couldn't see the whole set of changes. Maybe I'll start looking at restyling/master too when I do these reviews.